schemaregistry/schemaregistry.md

```diff
@@ -0,0 +1,318 @@
+  - OPTIONAL
+  - Assigned by the server.
+
+#### updatedtimeutc
+
+- Type: `Timestamp`
+- Description: Instant when the schema group was last updated
+- Constraints:
+  - OPTIONAL
+  - Assigned by the server.
+
+## 2.2. Schema and Schema Version
+
+Conceptually, a schema is a description of a data structure. Since data
+structures evolve over time, the schema describing them will also evolve over
+time. Therefore, a schema often has multiple versions.
+
+For simple scenarios, the API allows for version management to be automatic and
+transparent. Whenever a schema is updated, a new version number is assigned and
+prior schema versions are retained. This specification does not mandate a
+retention policy, but implementations MAY retire and remove outdated schema
+versions.
+
+The latest available schema is always the default version that is retrieved when
+the URL to the schema is given without the version specified.
+
+A newer schema version might introduce breaking changes or it might only
+introduce careful changes that preserve compatibility. These strategies are not
+subject of this specification, but the API provides a conflict handling
+mechanism that allows an implementation to reject updates that do not comply
+with a compatibility policy, if one has been implemented.
+
+### 2.2.1. Multi-format schema groups
+
+If the schema format has not been restricted at the schema group level, each
+schema MAY have its own format. This choice is mutually exclusive. If a format
+has been defined for the group, schemas in the group MUST use that format. 
+
+### 2.2.2. Schema attributes
+
+As per the definition above, the schema object is a collection of versions
+of schema documents. An implementation MAY add further attributes.
+
+#### id
+
+- Type: `String`
+- Description: Identifies the schema.
+- Constraints:
+  - REQUIRED
+  - MUST be a non-empty string
+  - MUST conform with RFC3986/3.3 `segment-nz-nc` syntax
+  - MUST be unique within the scope of the schema registry
+- Examples:
+  - myschema
+  - my-schema
+
+#### format
+
+- Type: `String`
+- Description: Defines the format of this schema. The formats supported by an
+  implementation are specific to the implementation. If the format has been omitted
+  for the schema group, the format MUST be specified at the schema-level. This
+  specification does not mandate support for particular formats.
+- Constraints:
+  - REQUIRED if not set at the schema group level.
+  - MUST be null (not present) if set at the schema group level.
+  - MUST be a non-empty string, if present.
+- Examples:
+  - "avro"
+  - "json-schema"
+  - "xsd11"
+
+#### description
+
+- Type: `String`
+- Description: Explains the purpose of the schema.
+- Constraints:
+  - OPTIONAL
+- Examples:
+  - "This schema describes the toppings for a Pizza."
+
+  #### createdtimeutc
+
+- Type: `Timestamp`
+- Description: Instant when the schema was added to the registry.
+- Constraints:
+  - OPTIONAL
+  - Assigned by the server.
```

Comment:
as above, I think we need a MUST here specifying the format/syntax - or define "Timestamp" in some kind of "data types" section